core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -284,6 +284,14 @@ class CityInfoReligionManager {
         }
         return addedPressure
     }
+
+    fun isProtectedByInquisitor(): Boolean {
+        for (tile in cityInfo.getCenterTile().neighbors)
+            if (tile.civilianUnit?.name == "Inquisitor")
+                return true
+        if (cityInfo.getCenterTile().civilianUnit?.name == "Inquisitor") return true
```

Comment:
This still uses the name of a unit instead of the unique, moddability :)

Reply:
Damn, next time I am checking each and every use of uniques